Core/MIPS/MIPSIntVFPU.cpp

```diff
@@ -1562,7 +1565,7 @@ namespace MIPSInt
 		int vd = _VD;
 		int vs = _VS;
 		int vt = _VT;
-		int ins = (op >> 23) & 7;
+		int ins = (op >> 23) & 3;
```

Comment:
FWIW, I think this is a good change, but it couldn't have actually caused problems because MIPSTables guarantees `& 4` will never reach here.

Reply:
Yup, I figured so. Still good to silence the analyzer which isn't smart enough to go that deep.